tests/conftest.py

```diff
@@ -11,3 +11,48 @@
 import pytest  # pylint: disable=unused-import
 
 pytest_plugins = ['aiida.manage.tests.pytest_fixtures']  # pylint: disable=invalid-name
+
+
+@pytest.fixture()
+def non_interactive_editor(request):
+    """Fixture to patch click's `Editor.edit_file`.
+
+    In `click==7.1` the `Editor.edit_file` command was changed to escape the `os.environ['EDITOR']` command. This was
+    breaking our tests since were abusing this system to define an automated vim command in order to make an interactive
+    command non-interactive. However, the escaping made the command and arguments understood as a single command and
+    so bash could not find it. Here we patch the method to remove the escaping of the editor command.
```

Comment:
```suggestion
    In `click==7.1` the `Editor.edit_file` command was changed to escape the `os.environ['EDITOR']` command.
    We are currently abusing this variable to define an automated vim command in order to make an interactive
    command non-interactive, and escaping it makes bash interpret the command and its arguments as a single command instead.
    Here we patch the method to remove the escaping of the editor command.
```

Reply:
Why is it enough to define this as a fixture? Is this relevant just for testing?
Perhaps I haven't fully grasped the problem here...

Reply:
I am pretty sure yeah, because I don't think we explicitly pass arguments to a file editor anywhere else in the code. But cannot be 100% sure

Reply:
Ok. Just checking whether you are still planning to accept the changes, which are meant describe the status quo rather than what changed.